.semaphore/cp_dockerfile_build.yml

```diff
@@ -25,7 +25,6 @@ global_job_config:
     commands:
       - checkout
       - sem-version java 8
-      - sem-version python 3.7
```

Comment:
The scripts depend on python 3.7. I think that might be the reason why build is failing